src/main/kotlin/com/d3/chainadapter/main.kt

```diff
@@ -26,11 +26,19 @@ const val CHAIN_ADAPTER_SERVICE_NAME = "chain-adapter"
 //TODO Springify
 fun main(args: Array<String>) {
     val chainAdapterConfig =
-        loadRawLocalConfigs("chain-adapter", ChainAdapterConfig::class.java, "chain-adapter.properties")
+        loadRawLocalConfigs(
+            "chain-adapter",
+            ChainAdapterConfig::class.java,
+            "chain-adapter.properties"
+        )
 
     val irohaCredential = chainAdapterConfig.irohaCredential
     ModelUtil.loadKeypair(irohaCredential.pubkeyPath, irohaCredential.privkeyPath).map { keyPair ->
         createLastReadBlockFile(chainAdapterConfig)
+        val lastReadBlockProvider = FileBasedLastReadBlockProvider(chainAdapterConfig)
+        if (chainAdapterConfig.dropLastReadBlock)
```

Comment:
how about 
```
val lastReadBlockProvider = if (chainAdapterConfig.dropLastReadBlock){ FileBasedLastReadBlockProvider(chainAdapterConfig)
} else {
StubLastReadBlockProvider() // just returns 0
}
```

or it does not make any sense?

Reply:
There is no need in StubLastReadBlockProvider, need only set zero on startup in some cases.

Reply:
Could be used in tests as well